deploy/operator.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: |
+    apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+    kind: ControllerManagerConfig
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: 127.0.0.1:8080
+    webhook:
+      port: 9443
+    leaderElection:
+      leaderElect: true
+      resourceName: 08db0feb.percona.com
+kind: ConfigMap
+metadata:
+  name: percona-server-for-mysql-operator-config
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: percona-server-for-mysql-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: percona-server-for-mysql-operator
+  template:
+    metadata:
+      labels:
+        control-plane: percona-server-for-mysql-operator
+    spec:
+      containers:
+      - args:
+        - --leader-elect
+        command:
+        - /usr/local/bin/percona-server-mysql-operator
+        env:
+        - name: WATCH_NAMESPACE
```

Comment:
Should we support cluster wide from very beginning or not? Also please create the bundle.yaml file

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/12/commits/274cac3208f0d261d435eca72f7d7058f98fbf4d